docs/src/.vuepress/config.js

```diff
@@ -246,10 +246,16 @@ module.exports = {
         text: 'Starter Kits',
         link: '/kits/',
         collapsible: true,
-        children: [{
-          text: 'Svelte',
-          link: '/kits/svelte/'
-        }]
+        children: [
+          {
+            text: 'Svelte',
+            link: '/kits/svelte/'
+          },
+          {
+            text: 'React',
+            link: '/kits/react/'
+          }
+        ]
```

Comment:
If you could swap React and Svelte in the list, we are trying to keep all the list sorted by Alphanumerics.

Reply:
Done!